modules/apps/segments/segments-web/test/js/utils/odata-util.es.js

```diff
@@ -0,0 +1,183 @@
+import * as Util from '../../../src/main/resources/META-INF/resources/js/utils/odata-util.es';
```

Comment:
Is there any way to co-locate the unit tests with the src files? Or do we have to separate them because of how it is compiled and deployed?

Reply:
This was to keep it consistent with the other portlet structures which had jest tests as well. For this ticket I'll keep it the same, but I'll see what Chema's thoughts are if we can move the tests elsewhere and make the changes in another ticket if we can.